test/integration/params-validations-test.js

```diff
@@ -22,8 +22,26 @@ describe('params validations', () => {
 
   it('request error', () => {
     const github = new GitHub({
+      baseUrl: 'https://127.0.0.1:8' // port: 8 // officially unassigned port. See https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers
+    })
+
+    return github.orgs.get({org: 'foo'})
+
+      .catch(error => {
+        expect(error.toJSON()).to.deep.equal({
+          code: 500,
+          message: 'request to https://127.0.0.1:8/orgs/foo failed, reason: connect ECONNREFUSED 127.0.0.1:8',
+          status: 'Internal Server Error'
+        })
+      })
+  })
+
+  it('coverage', () => {
+    const github = new GitHub({
+      baseUrl: null,
       host: '127.0.0.1',
-      port: 8 // officially unassigned port. See https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers
+      port: '8',
+      protocol: 'https'// port: 8 // officially unassigned port. See https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers
     })
```

Comment:
remove the coverage test here, we should cover it all in `deprecations-test.js`

Reply:
Should i move this entire test to deprecations-test.js ?

Reply:
you can remove it entirely, we don’t need it, you’ll just need to adjust the deprecations test and the implementation a bit, see my other comments